examples/t8code_2d_dgsem/elixir_advection_restart.jl

```diff
@@ -0,0 +1,43 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# create a restart file
+
+elixir_file = "elixir_advection_extended.jl"
+restart_file = "restart_000021.h5"
```

Comment:
```suggestion
restart_file = "restart_000000021.h5"
```
Due to the breaking change in v0.8.